user_guide/writing_scenarios.rst

```diff
@@ -280,6 +280,19 @@ The scenario that is actually run is:
       # <left> replaced with 7:
       Then I should have 7 cucumbers
 
+Multiline Arguments
+-------------------
+
+The one line `steps`_ let Behat extract small strings from your steps
+and receive them in your step definitions. However, there are times when you
+want to pass a richer data structure from a step to a step definition.
+
+This is what multiline step arguments are designed for. They are written on
+lines immediately following a step and are passed to the step definition
+method as the last argument.
+
+Multiline step arguments come in two flavours: `tables`_ or `pystrings`_.
+
 Tables
 ------
```

Comment:
those should probably even be a different heading level, so that they are sub-sections of the section about multiline arguments

Reply:
Good shout, I'll try to add that later 😁

Reply:
I did it in https://github.com/Behat/docs/pull/163

Reply:
Just 1 minute before I created another PR https://github.com/Behat/docs/pull/164 😆 I closed mine